posthog/models/person/person.py

```diff
@@ -187,6 +179,23 @@ class Meta:
     version: models.BigIntegerField = models.BigIntegerField(null=True, blank=True)
 
 
+class PendingPersonOverride(models.Model):
+    id = models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name="ID")
+    team_id = models.BigIntegerField()
+    old_person_id = models.UUIDField()
+    override_person_id = models.UUIDField()
+    oldest_event = models.DateTimeField()
+
+
+class FlatPersonOverride(models.Model):
+    id = models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name="ID")
+    team_id = models.BigIntegerField()
```

Comment:
This differs from `PersonOverride`, which defines `team_id` as a foreign key to team. (`PendingPersonOverride` uses a plain big integer here as well.)

Reply:
Maybe there's nothing useful to be said (that doesn't become a book), but I wonder if this and `PendingPersonOverride` could get a small comment about what they're for. Mostly because the comment on `PersonOverride` was helpful to me when I came in knowing nothing.

Reply:
Also, just putting it here: Do we really not want any indexes on `FlatPersonOverride`? I need to read all the worker code still, but I figure an index on `team_id` may reduce rows scanned when you have to do updates, or something? I know the table shouldn't be contended, but I wonder how the queries will work when there are like 100 million rows in here. (Could be totally wrong.)

Reply:
Coming back, it seems like `(team_id, override_person_id)` would be useful?

Reply:
> Maybe there's nothing useful to be said (that doesn't become a book), but I wonder if this and `PendingPersonOverride` could get a small comment about what they're for. Mostly because the comment on `PersonOverride` was helpful to me when I came in knowing nothing.

Good point, will add this here.

> Also, just putting it here: Do we really not want any indexes on `FlatPersonOverride`? I need to read all the worker code still, but I figure an index on `team_id` may reduce rows scanned when you have to do updates, or something? I know the table shouldn't be contended, but I wonder how the queries will work when there are like 100 million rows in here. (Could be totally wrong.) […] Coming back, it seems like `(team_id, override_person_id)` would be useful?

Also a really good point, I had been thinking so much about whether or not we want/need constraints here that I totally lapsed on indexes for performance's sake -- thanks for pointing it out! I agree, the one you mentioned here seems like it could be useful.

Reply:
Added the index, as well as a couple explanatory comments here.